reducers/views/rhs.js

```diff
@@ -115,6 +115,8 @@ function searchTerms(state = '', action) {
 function pluggableId(state = '', action) {
     switch (action.type) {
     case ActionTypes.UPDATE_RHS_STATE:
+    case ActionTypes.SELECT_POST:
+    case ActionTypes.SELECT_POST_CARD:
```

Comment:
I'm surprised there is no value for thread in the `RHSState`. It should be ideal that the state would be updated to that when it is opened to reply. Not loving the mix of "unrelated" `SELECT_POST` and `SELECT_POST_CARD` actions here.

Reply:
Agree, `SELECT_POST` and `SELECT_POST_CARD` should be moved under `RHSStates`. Unfortunately that requires a bigger refactor, not worth it for this bug. Have to see if it is how it is for legacy reasons or some other reason, if there is no good reason we should just create a HW ticket to clean it up.

@hmhealey might have more context on this.